### PR TITLE
[ERP-2252] Updated README, added three tables to explain each of the CSVs' values

### PR DESCRIPTION
**Pull request by NikuPAN**
Updated README, added three tables to explain each of the CSVs' values

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,16 +38,43 @@ Data for current incident is from [incidents.csv](incidents.csv). There are two
...
 
 ### Past Incidents
 Data for past incident is from [past_incidents.csv](past_incidents.csv). When you open the file, you will either see an empty file means there are no past incidents history, and if you see date, system and reason means there are incidents happened in the past.
 
 To update [past_incidents.csv](past_incidents.csv), if the file is empty and you want to add a history, add a line with the following format (values separated with a comma): `2024-07-20, HPC-FS, scheduled maintenance`.
 
+#### Past Incidents Data Values Descriptions
+| # | Value Name   | Description                     | Data Type   | Example Value         |
+|---|--------------|---------------------------------|-------------|-----------------------|
+| 1 | Date         | The date of outage reported     | String/Date | 2024-07-20            |
```
Comment (ryan-bennett) [inline README.md]: ISO 8601 Date for Data Type

File: README.md [hunk 2]
```diff
@@ -38,16 +38,43 @@ Data for current incident is from [incidents.csv](incidents.csv). There are two
 
 To update [incidents.csv](incidents.csv), In case any incident happens, remove the line "No ongoing incidents" and replace with the following format (values separated with a comma): `PBS Server, 2024-07-02T15:53:42+1000, Disrupted`. 
 
+#### Current Incidents Data Values Descriptions
+| # | Value Name   | Description                            | Data Type   | Example Value            |
+|---|--------------|----------------------------------------|-------------|--------------------------|
+| 1 | Service Name | Name of the service disrupted          | String      | PBS Server               |
+| 2 | Reported     | The date and time outage were reported | String/Date | 2024-07-02T15:53:42+1000 |
```
Comment (ryan-bennett) [inline README.md]: ISO 8601 Date Time for Data Type

File: README.md [hunk 3]
```diff
@@ -38,16 +38,43 @@ Data for current incident is from [incidents.csv](incidents.csv). There are two
...
 
 ## Updating Planned Maintenance Events
-When adding planned maintenance events, edit [planned_maintenance.csv](planned_maintenance.csv) and add a new row with the following format (values separated with a comma): `HPC-FS, 2024-01-31T09:00:00+1000, 2024-01-31T12:00:00+1000, Service will be unavailable`. 
+When adding planned maintenance events, edit [planned_maintenance.csv](planned_maintenance.csv) and add a new row with the following format (values separated with a comma): `HPC-FS, 2024-01-31T09:00:00+1000, 2024-01-31T12:00:00+1000, Service will be unavailable`.
+
+#### Planned Maintenance Data Values Descriptions
+| # | Value Name       | Description                                        | Data Type   | Example Value               |
+|---|------------------|----------------------------------------------------|-------------|-----------------------------|
+| 1 | Service Name     | Name of the service will be affected               | String      | HPC-FS                      |
+| 2 | Date & Time From | The date and time when this maintenance starts     | String/Date | 2024-01-31T09:00:00+1000    |
```
Comment (ryan-bennett) [inline README.md]: ISO 8601 Date Time for Data Type

File: README.md [hunk 4]
```diff
@@ -38,16 +38,43 @@ Data for current incident is from [incidents.csv](incidents.csv). There are two
...
+| 3 | Date & Time To   | The date and time when this maintenance finishes   | String/Date | 2024-01-31T12:00:00+1000    |
```
Comment (ryan-bennett) [inline README.md]: ISO 8601 Date Time for Data Type

**Review by ryan-bennett — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (NikuPAN) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (NikuPAN) [reply README.md]: Updated

File: README.md (hunk 4 above)
Comment (NikuPAN) [reply README.md]: Updated

File: README.md (hunk 1 above)
Comment (NikuPAN) [reply README.md]: Updated

**Review by ryan-bennett — APPROVED** [review]

**Pull request merged**